tests/manage/mcg/test_bucket_deletion.py

```diff
@@ -4,31 +4,54 @@
 
 from ocs_ci.framework import config
 from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.utility.utils import check_if_executable_in_path
 
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.skipif(condition=True, reason="MCG is not deployed")
 @pytest.mark.filterwarnings(
     'ignore::urllib3.exceptions.InsecureRequestWarning'
 )
+@pytest.mark.skipif(
```

Comment:
define also marker form aws_platform_required and reuse it here and I see already was used somewhere above once.